homeassistant/components/solax/__init__.py

```diff
@@ -1 +1,26 @@
 """The solax component."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from ...const import Platform
+from .const import DOMAIN
+
+PLATFORMS = [Platform.SENSOR]
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up the sensors from a ConfigEntry."""
+    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
```

Comment:
Missing return typing

```suggestion
async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```